public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -0,0 +1,252 @@
+include ../_util-fns
+
+:marked
+  # Http
+  
+  Our application has become a huge success and our stakeholders have already expanded the vision to include integration with a hero api. 
+  
+  The current solution limits us to a fixed set of heroes, but integration with the api will allow us to load heroes dynamically from a database. We will also be able to add, edit and delete heroes. 
+  
+  In this chapter we will show how to integrate the http based api with our existing application.
```

Comment:
Suggest `In this chapter we will connect our Angular 2 services to make http calls to our new api.`